pwd_guess.py

```diff
@@ -1007,6 +1063,15 @@ def retrain_classification(self, preprocessor, serializer):
         self.pwd_list = preprocessor
         self.train(serializer)
 
+    def write_log(self, names, logs, batch_no):
+        assert(self.callback)
+        for name, value in zip(names, logs):
+            summary = tf.Summary()
+            summary_value = summary.value.add()
+            summary_value.simple_value = value
+            summary_value.tag = name
+            self.callback.writer.add_summary(summary, batch_no)
+            self.callback.writer.flush()
```

Comment:
Could we only flush the logs on the chunk_print_interval so that we avoid flushing every iteration (and spending the time to do it)?